### DISCUSSION TITLE
How can new a record for a relation table?

### DISCUSSION THREAD

**Discussion by joisun** [prisma/prisma#16682] [Q&A]
Hi there!
I'm trying to build a blog system, and there are serveral related table to Article creation. like Category, Tags, and Column. 
And the problem I met It's I'm new to use orm like prisma.  I'm not familiar to it.
for detail:
```js
  create(userid: string, createAdminArticleDto: CreateAdminArticleDto) {
    const {
      title,
      content,
      isbanner,
      description,
      cover,
      password,
      visible,
      tags,
      column,
      category,
    } = createAdminArticleDto;

    this.prisma.article.create({
      data: {
        title,
        content,
        authorId: userid,
        banner: isbanner ? 1 : 0,
        description,
        cover,
        password,
        visible,
        columns: {
          connect: {

            // artid_colid: {
            //   colid: column,
            //   artid: '',
            // },
          },
        },
```
Qustion1: What is the hint field `artid_colid` ?
Qustion2: How to exactly  write the code about add a new record to ColumnsOnArticles/TagsOnArticles/CategoriesOnArticles relation table when I trying to create a new article like above ?
(column/category table  content is predifined before we create a new article, but tag table's record may not predefined, user can add a new tag while creating a new article the same time)

This problems blocked me for twos days, help!!!! and HUGE THANKS!!

**Comment by nurul3101** [comment]
Hey @jaycethanks 👋 
Can you share your schema file so I can help write the create query?

The hint `artid_colid` would indicate that columns model has an id field which is the combination of both artid and colid fields, so to connect a columns, you would need to pass values of artid as well as colid.

**Reply by pantharshit00** [reply]
👋 Hello @jaycethanks,  did you perhaps have a chance to check @nurul3101's recommendation? Let us know if you still have any questions! 
 
Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
